.appveyor.yml

```diff
@@ -71,7 +77,7 @@ test_script:
   - pip install %NP_TEST_DEP%
   - pip install -r %REPO_DIR%\requirements.txt
   # Install the generated wheel package to test it
-  - pip install --pre --no-index --find-links .\%REPO_DIR%\dist %PKG_NAME%
+  - pip install --no-index --find-links .\%REPO_DIR%\dist %PKG_NAME%
```

Comment:
Why remove pre? I think the idea is that on master, this should be enabled, and on the branches it can be removed.

Reply:
we are already testing with --pre on the official repo on every PR. do you want this here too? I can add pre as an environment variable, but will you rememeber to remove it during release?

Reply:
I believe this was added precisely because something that was passing in the main repo was not passing here when it came time to release. The correct procedure would be to keep it here, then then add a note in the release instructions to remove it.

Reply:
??? ok, if you think that is best.

Reply:
I really believe much of this manual dancing can be done automatically, but requires us to ask travis and appveyor to simply upload wheels after they are done. In addition, much of this complexity is kinda created by supporting multiple branches, which I also think complicate the release model.

I'm not so sure scikit-image is core enough to many packages that supporting multiple minor versions is so important. A linear release model would streamline the process quite a bit.

Reply:
We're usually fairly linear. This situation was brought on by the long term release to support Python 2, which is now history.